### PR TITLE
Upgrade Admin User-Management Table

### PR DESCRIPTION
**Pull request by GodGreg**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -31,7 +31,8 @@
     "react-hooks/exhaustive-deps": "warn",
     "no-useless-rename": "warn",
     "prefer-const": "warn",
-    "no-var": "warn"
+    "no-var": "warn",
+    "no-console": ["warn", { "allow": ["error", "info"] }]
```
Comment (ahmedhamodi) [inline .eslintrc:35]: Nice!

File: src/Authentication/logincards/passwordresetemailcard/PasswordResetEmailCard.js [hunk 2]
```diff
@@ -69,7 +69,7 @@ function PasswordResetEmailCard({ onSubmit, backToLogin }) {
       await firebaseApp.auth().sendPasswordResetEmail(values.email);
       onSubmit();
     } catch (e) {
-      console.log(e);
+      //console.error(e);
```
Comment (ahmedhamodi) [inline src/Authentication/logincards/passwordresetemailcard/PasswordResetEmailCard.js:72]: Not sure if you wanted to comment this out, but I'll just point it out so you are aware.

File: src/Components/AllCandidates/AllCandidatesTable.jsx [hunk 3]
```diff
@@ -30,6 +30,7 @@ function AllCandidatesTable(props) {
   ];
   return (
     <div>
+      <button></button>
```
Comment (ahmedhamodi) [inline src/Components/AllCandidates/AllCandidatesTable.jsx:33]: Empty button tags?

File: src/requests/delete.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ async function deleteUserAPI(userId) {
     method: "DELETE"
   });
   if (response.status !== 204) {
-    console.log(`Error with deleting user with userid = ${userId}`);
+    //console.error(`Error with deleting user with userid = ${userId}`);
```
Comment (ahmedhamodi) [inline src/requests/delete.js:14]: Another commented out console error.

File: src/Components/AllCandidates/AllCandidates.jsx [hunk 5]
```diff
@@ -169,20 +176,31 @@ function AllCandidates({ history }) {
...
+  //   allUsers.value.forEach((user) => {
+  //     const exist = user.programs.some((p) => {
+  //       if (p.id == program) {
+  //         console.log(p.id);
+  //         return true;
+  //       }
+  //     });
+  //     console.log(exist);
+  //   });
+  // }
```
Comment (ahmedhamodi) [inline src/Components/AllCandidates/AllCandidates.jsx:202]: Intentional?

**Review by ahmedhamodi — APPROVED** [review]
LGTM. Some small comments if you want to ensure all that was intentional.

File: src/Authentication/logincards/passwordresetemailcard/PasswordResetEmailCard.js (hunk 2 above)
Comment (GodGreg) [reply src/Authentication/logincards/passwordresetemailcard/PasswordResetEmailCard.js:72]: Yeup. It is frontend so we don't need that displaying. But it is a commonly used error that we should log in the future.

Comment (GodGreg) [reply src/Authentication/logincards/passwordresetemailcard/PasswordResetEmailCard.js:72]: Wanted to keep it commented so i could find all the console.errors later for the frontend

File: src/Components/AllCandidates/AllCandidatesTable.jsx (hunk 3 above)
Comment (GodGreg) [reply src/Components/AllCandidates/AllCandidatesTable.jsx:33]: Good catch! I removed them

File: src/Components/AllCandidates/AllCandidates.jsx (hunk 5 above)
Comment (GodGreg) [reply src/Components/AllCandidates/AllCandidates.jsx:202]: Should be removed.

**Pull request merged**